common/changes/@bentley/webgl-compatibility/2.10-audit-fix_2021-06-03-23-14.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
```

Comment:
I don't think most of these are needed. You'll want to delete these and run `rush change -b release/2.10.0`.